### PR TITLE
Umbraco forms validation update

### PR DESCRIPTION
**Pull request by nielslyngsoe**
Made corrections to the validation description to accommodate that we now that a different validation framework default with Umbraco Forms. But jQuery is still an option if you include it manually.
Notice this feature is not released yet, so we need to wait to merge this in until that happens.

### CODE REVIEW

File: Add-ons/UmbracoForms/Developer/Prepping-Frontend/index.md [hunk 1]
```diff
@@ -4,14 +4,29 @@ versionFrom: 7.0.0
 
 # Preparing your frontend
 
-In order for Umbraco Forms to work correctly, Umbraco Forms needs three (3) client dependencies.
+In order for Umbraco Forms to work correctly, Umbraco Forms need you to include some client dependencies.
```
Comment (sofietoft) [inline Add-ons/UmbracoForms/Developer/Prepping-Frontend/index.md:7]:
```suggestion
In order for Umbraco Forms to work correctly, you need to include some client dependencies.
```

File: Add-ons/UmbracoForms/Developer/Prepping-Frontend/index.md [hunk 2]
```diff
@@ -4,14 +4,29 @@ versionFrom: 7.0.0
...
+
+With latest version of Umbraco Forms, we provide a method that will output the script tags containing the dependencies. Dne with this razor method:
```
Comment (sofietoft) [inline Add-ons/UmbracoForms/Developer/Prepping-Frontend/index.md:9]:
```suggestion
Use the following Razor method to output script tags containing the dependencies:
```

File: Add-ons/UmbracoForms/Developer/Prepping-Frontend/index.md [hunk 3]
```diff
@@ -4,14 +4,29 @@ versionFrom: 7.0.0
...
+
+```html
+<head>
+    @Html.RenderUmbracoFormDependencies()
+</head>
+```
+
+All dependencies originates from your UmbracoForms instalation, no external references.
```
Comment (sofietoft) [inline Add-ons/UmbracoForms/Developer/Prepping-Frontend/index.md:17]:
```suggestion
All dependencies originates from your Umbraco Forms installation, which means that no external references are needed.
```

File: Add-ons/UmbracoForms/Developer/Prepping-Frontend/index.md [hunk 4]
```diff
@@ -4,14 +4,29 @@ versionFrom: 7.0.0
...
+
+If you like to use jQuery instead, you will instead have to manually add those. Without using the above. Jump the the section 'Use jQuery' to learn more.
```
Comment (sofietoft) [inline Add-ons/UmbracoForms/Developer/Prepping-Frontend/index.md:19]:
```suggestion
If you like to use jQuery instead, you can manually add those. Without using the above, jump the 'Use jQuery' section below to learn more.
```

**Comment by nielslyngsoe** [comment]
@sofietoft can latest Umbraco Forms run v7? if so yes. but it's only for the latest Umbraco Forms, when it comes out. It's not out yet.

**Comment by jmayntzhusen** [comment]
Since we can't currently version the docs based on Forms versions but only Umbraco versions I'd recommend we add some warnings throughout these articles explaining that jQuery is the way to go for Forms versions x to x, and you can choose between jQuery and .NET validation from version x..

Otherwise we will see a lot of confusion by people not on the latest Forms version - and this may encourage some to upgrade 😁

**Comment by sofietoft** [comment]
Hi there!
I have an important message for you, regarding this PR:

Yesterday we tried to do some cleaning up and force-pushed to the master branch which now causes some problems with existing forks and local clones. 
We have fixed up all the open PRs so that they are still mergeable now.

With that said, if you've been working on a **fork of the repo** or in a **local clone**, those will not work anymore.
If you wish to push any new changes to this PR, you will need to delete your local clone or fork and create fresh ones.

If you have any local work in progress, then you copy it over from your old fork or clone, following these steps:

**Forks**
1. In the old clone of the old fork, switch to the branch you were working on
2. In the new clone of the new fork, create a new branch with the same name as the old one
3. Copy all the affected files from the old clone to the new clone
4. Repeat steps 1-3 if you've been working with multiple branches
5. You can now safely delete your old fork

**Clones**
1. In the old clone, switch to the branch you were working on
2. Create and checkout a new branch on the new clone, using the same name as the old clone
3. Copy all the affected files from the old clone to the new clone
4. Repeat steps 1-3 if you've been working with multiple branches
5. Delete the old clone of the docs repo

And with that, you should be all set once more.

I do apologize for the inconvenience, and please let me know if you have any questions!
/Sofie

**Review by thomasandresen — COMMENTED** [review]
All look just fine!
Changes to forms and starter kit repository has been merged into dev branch, so the doc should be only be published once we release a new form version.

**Pull request merged**